src/draw/drawLineOnIjs.ts

```diff
@@ -40,13 +40,15 @@ export function drawLineOnIjs(
   checkProcessable(newImage, 'drawLine', {
     bitDepth: [8, 16],
   });
+  const numberChannels = Math.min(newImage.channels, color.length);
 
   if (from.column === to.column && from.row === to.row) {
+    for (let channel = 0; channel < numberChannels; channel++) {
+      newImage.setValue(from.column, from.row, channel, color[channel]);
+    }
     return newImage;
   }
```

Comment:
I think you can just remove the `if` and let the library handle this case?

Reply:
yes okey no problem

Reply:
yes it handle the case